packages/Counter/src/Counter.styles.js

```diff
@@ -43,7 +43,7 @@ const counterColorStyles = {
   `,
 };
 
-const indicatorStyles = css`
+const indicatorStyles = `
```

Comment:
I think it's a good idea to keep the `css` helper here, mainly because it will improve the syntax highlighting in our editors.